src/components/Extensibility/ExtensibilityDetails.js

```diff
@@ -119,6 +120,44 @@ export const ExtensibilityDetailsCore = ({
             ]
           : []
       }
+      customStatusColumns={
```

Comment:
We still render empty div even if `customStatusColumns` are empty
<img width="1070" alt="Screenshot 2024-06-07 at 09 20 13" src="https://github.com/kyma-project/busola/assets/20790348/29a316d4-eb4a-40ed-b392-205ade0b5d37">